terraform/alerting/alerts.tf

```diff
@@ -50,6 +50,41 @@ resource "google_monitoring_alert_policy" "CloudSchedulerJobFailed" {
   ]
 }
 
+resource "google_monitoring_alert_policy" "probers" {
+  project = var.project
+
+  display_name = "HostDown"
+  combiner     = "OR"
+  conditions {
+    display_name = "Host is unreachable"
+    condition_monitoring_query_language {
+      duration = "300s"
```

Comment:
I think I'd prefer to be alerted sooner if a host is completely down. Can we alert on the 1min interval?